webpack.dev.js

```diff
@@ -33,16 +37,13 @@ module.exports = [
     mode: 'development',
     devtool: 'inline-source-map',
     devServer: configureDevServer(port),
-    plugins: [
-      new webpack.HotModuleReplacementPlugin(),
-    ],
-  }),
-  merge(common.liteConfig, {
-    mode: 'development',
-    devtool: 'inline-source-map',
-    devServer: configureDevServer(port + 1),
-    plugins: [
-      new webpack.HotModuleReplacementPlugin(),
-    ],
   }),
+  //merge(common.liteConfig, {
+  //  mode: 'development',
+  //  devtool: 'inline-source-map',
+  //  devServer: configureDevServer(port + 1),
+  //  plugins: [
+  //    new webpack.HotModuleReplacementPlugin(),
+  //  ],
+  //}),
```

Comment:
@jourdain why do we run two webpack configs when running `npm run build`?

Reply:
To have a trimmed down version of vtk.js but now that we have ESM, I don't see the need anymore.